rep-0149.rst

```diff
@@ -0,0 +1,1091 @@
+REP: 149
+Title: Package Manifest Format Three Specification
+Author: Dirk Thomas
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 11-Oct-2017
+
+Outline
+=======
+
+#. Abstract_
+#. Motivation_
+#. Rationale_
+#. `Data Representation`_
+#. Compatibility_
+#. Schema_
+#. References_
+#. Copyright_
+
+.. raw:: html
+
+  The current draft highlights significant changes compared to REP-0140 [2]_
+  in <font color="red">red</font>.
+  Trivial changes appear in <font color="green">green</font>.
+
+
+Abstract
+========
+
+.. raw:: html
+
+  <font color="green">
+
+This REP specifies the third ``package.xml`` format, replacing the ones
+specified in REP-0127 [1]_ and REP-0140 [2]_.
+It is relevant for packages using the catkin as well as the ament build system.
```

Comment:
I removed the sentence (2e7172fb5e658afa1d9e5e6a06fee73948ea962a) since most is not specifically "more complex" and the rest of the document should cover it already.